ReactApp/src/components/Editor.jsx

```diff
@@ -168,25 +174,31 @@ function Editor({ post }) {
 	// }
 
 	return (
-		<div className="editor__container">
-			<BlockEditorProvider
-				value={postBlocks}
-				onInput={onBlockEditorInput}
-				onChange={onBlockEditorChange}
-				settings={settings}
-			>
-				<BlockCanvas shouldIframe={false} height="auto" styles={styles}>
-					<div className="editor-visual-editor__post-title-wrapper">
-						{isEditorReady && <PostTitle ref={titleRef} />}
-					</div>
-					<BlockList />
-				</BlockCanvas>
-				{isEditorReady && <EditorToolbar />}
-
-				<Popover.Slot />
-				<EditorSnackbars />
-			</BlockEditorProvider>
-		</div>
+		hasLoadedPost && (
+			<div className="editor__container">
+				<BlockEditorProvider
+					value={postBlocks}
+					onInput={onBlockEditorInput}
+					onChange={onBlockEditorChange}
+					settings={settings}
+					useSubRegistry={false}
```

Comment:
This change was done in https://github.com/wordpress-mobile/GutenbergKit/pull/22 I'll update with `trunk` once it is merged.